build/toolchain/mac/BUILD.gn

```diff
@@ -263,10 +271,16 @@ mac_toolchain("ios_clang_x64") {
 mac_toolchain("clang_x64") {
   toolchain_cpu = "x64"
   toolchain_os = "mac"
-  prefix = rebase_path(llvm_bin_path, root_build_dir)
-  ar = "$prefix/llvm-ar"
-  cc = "$prefix/clang"
-  cxx = "$prefix/clang++"
+  if (!enable_bitcode) {
```

Comment:
We can avoid this for host builds. `x64`

Reply:
I get linking errors if I don't do this because of gen_snapshot